public_html/wp-content/plugins/pattern-directory/includes/pattern-post-type.php

```diff
@@ -130,16 +130,17 @@ function register_post_type_data() {
 		POST_TYPE,
 		'wpop_viewport_width',
 		array(
-			'type'              => 'number',
+			'type'              => 'string',
+			'pattern'           => '[0-9]+(px|%|rem|em|vw)',
```

Comment:
The `viewportWidth` when used in patterns [needs to be an integer](https://github.com/WordPress/gutenberg/tree/94db813bc53f52bd28268fd3f7b4daf7982109cb/packages/block-editor/src/components/block-preview#viewportwidth), so this value needs to stay a number.

Reply:
Ah, I guess I misread: https://github.com/WordPress/pattern-directory/issues/59#issuecomment-849039740.
It has to be an integer. I'm going to reverse out the unit support and for `px`s.

Reply:
Done in: 2d3c6dd